src/installer/installer.go

```diff
@@ -123,11 +123,18 @@ func (i *installer) InstallNode() error {
 			return err
 		}
 	}
+	//upload host logs and report log status before reboot
+	err = i.inventoryClient.HostLogProgressReport(ctx, i.Config.ClusterID, i.Config.HostID, models.LogsStateRequested)
```

Comment:
it is not needed here, you have as part of logsender

Reply:
I need it for the case where log sender is stuck or can not communicate with the service: We saw such cases in the triage